docs/framework/guides/architecture/ui-library.md

```diff
@@ -292,6 +292,71 @@ toolbar.on( 'execute', evt => {
 } );
 ```
 
+### Dropdowns
+
+#### Creating ListView dropdown with standard button
+
+```js
+import Model from '@ckeditor/ckeditor5-ui/src/model';
+
+import { addListToDropdown, createDropdown } from '@ckeditor/ckeditor5-ui/src/dropdown/utils';
+
+const items = [
+	new Model( {
+		label: 'Do Foo',
+		commandName: 'foo'
+	} ),
+	new Model( {
+		label: 'Do Bar',
+		commandName: 'bar'
+	} ),
+];
+
+const dropdownView = createDropdown( locale );
+
+addListToDropdown( dropdownView, items );
```

Comment:
According to the [API](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/api/module_ui_dropdown_utils.html), `items` supposed to be a `Collection`.